js/domUpdates.js

```diff
@@ -1,59 +1,99 @@
-const players = [];
-const clues = [];
-const rounds = [];
-let game;
+const jeopardy = {
+  players: [],
+  clues: [],
+  rounds: [],
+  game: {},
+  categoryNames: {
+    unitedStatesHistory: 'United States History',
+    lifeSciences: 'Life Sciences',
+    publicHealth: 'Public Health',
+    educationJargon: 'Education Jargon',
+    nameThatBoardGame: 'Name That Board Game',
+    americanLiterature: 'American Literature',
+    biographies: 'Biographies',
+    americanCities: 'American Cities',
+    food: 'Food',
+    cableTV: 'Cable TV'
+  }
+};
 
-$('.input--submit').on('click', createPlayers);
-$('.main').on('click', presentClue);
+$('.input--submit').on('click', instantiatePlayers);
 
 function presentClue(event) {
-  console.log(event.target)
-  let colRow = event.target.dataset.colrow;
-  console.log(colRow);
-  let selectedClue = rounds[0].clues[parseInt(colRow)];
-  console.log(selectedClue)
-  alert(selectedClue.question);
+  if (event.target.innerHTML !== '') {
+    const id = event.target.dataset.id;
+    const selectedClue = jeopardy.rounds[0].clues[parseInt(id)];
+    if (selectedClue.dailyDouble === true) {
+      alert('DAILY DOUBLE!\n' + selectedClue.question);
+    } else {
+      alert(selectedClue.question);
```

Comment:
There is a big petition going on to deprecate `alerts` because so many apps abuse them. Not gunna happen any time soon, but you should eventually replace this with a little pop-up modal window or overlay or something when you have time.

Reply:
We will definitely do that.